rcl/package.xml

```diff
@@ -35,6 +35,7 @@
   <test_depend>rmw</test_depend>
   <test_depend>rmw_implementation_cmake</test_depend>
   <test_depend>launch</test_depend>
+  <test_depend>launch_testing</test_depend>
```

Comment:
Just a note: this adds another case of a circular repository dependency (same as for the test dependency on launch).

Reply:
This doesn't make that case any worse, but it is just an actual circular dependency. The testing code will likely need to be split into ROS and non-ROS parts like launch itself is, otherwise anything at or below rclcpp/rclpy will not be able to use any of it.